Source/SpectrumAnalysis/KTSequentialTrackFinder.hh

```diff
@@ -18,60 +18,67 @@
+#include "KTSparseWaterfallCandidateData.hh"
 #include "KTDiscriminatedPoints1DData.hh"
+#include "KTDiscriminatedPoint.hh"
 
-#include <iostream>
 #include <set>
 
 
 namespace Katydid
 {
     /*!
      @class KTSeqTrackFinder
-     @author E. Christine
+     @author C. Claessens
 
-     @brief Implementation of a slightly modified version of Dan Furse's algorithm
+     @brief Implementation of Dan Furse's algorithm with some modifications
 
      @details
      Collects points on a linear track
 
-     Configuration name: "dans-track-finding-algorithm"
+     Configuration name: "sequential-track-finder"
 
      Available configuration values:
-     - "snr-threshold-power": discrimination snr for point candidates
      - "min-frequency": minimum allowed frequency (has to be set)
      - "max-frequency": max allowed frequency (has to be set)
      - "min-bin": can be set instead of min frequency
      - "max-bin": can be set instead of  max frequency
-     - "trimming-factor": before a line is converted to a track its edges get trimmed. If the last or first line point power is less than the trimming-factor times the power threshold of the points frequency bins in the power spectrum slice, these points get cut off the line
-     - "line-power-radius": the power that is assigned to a line point is the sum of the power_spectrum[point_bin - line_width: point_bin + line_width]
+     - "trimming-threshold": before a line is converted to a sparse waterfall candidate its edges get trimmed. If the last or first line point snr is less than the trimming-threshold, they get removed
+     - "line-power-radius": only valid for disc1d-ps slot. the power that is assigned to a line point is the sum of the power_spectrum[point_bin - line_width: point_bin + line_width]
      - "time-gap-tolerance": maximum gap between points in a line (in seconds)
-     - "minimum-line-distance": if a point is less than this distance (in bins) away from the last point it will be skipped
-     - "search-radius": before a point is added to a line, the weighted average of the points frequency neighborhood (+/- search-radius in bins) is taken and the point updated until the frequency converges
-     - "converge-delta": defines when convergence has been reached (in bins)
+     - "minimum-line-distance": requires some thought for disc1d-slot!!! For disc1d-ps slot: if a point is less than this distance (in bins) away from the last point it will be skipped
+     - "search-radius": for disc1d-ps slot: before a point is added to a line, the weighted average of the points frequency neighborhood (+/- search-radius in bins) is taken and the point updated until the frequency converges
+     - "converge-delta": for disc1d-ps slot: defines when convergence has been reached (in bins)
      - "frequency-acceptance": maximum allowed frequency distance of point to an extrapolated line (in Hz)
      - "slope-method": method to update the line slope after point collection
```

Comment:
Documentation should list the options.  You could put a bulleted list below the set of parameters.